### ISSUE TITLE
4337 example

### ISSUE THREAD

**Issue by aodhgan** [Destiner/eip-7702#1]
curious how the 4337 example is relevant to 7702: https://eip7702.io/examples#sending-a-user-operation

seems like normal 4337 userop & transaction construction. what am i missing?

**Comment by Destiner** [comment]
You're right, it's exactly the same as with a smart contract account. Added a note to clarify
